src/_guides/language/effective-dart/usage.md

```diff
@@ -1083,15 +1081,14 @@ class Point {
 {% endprettify %}
 
 
-### DON'T use `new` in code that doesn't need to support Dart 1.
+### DON'T use `new`.
 
 Dart 2 makes the `new` keyword optional. Even in Dart 1, its meaning was never
 clear because factory constructors mean a `new` invocation may still not
 actually return a new object.
 
 The language still permits `new` in order to make migration less painful, but
-consider it deprecated and remove it from your code as soon as you no longer
-need to run on older Dart tools.
+consider it deprecated and remove it from your code.
```

Comment:
IDEs still don't have good support for generating code that doesn't have `new`. Perhaps wait on this guideline? Make it CONSIDER?

/cc @wmleler @filiph @kevmoo

Reply:
I'm not an expert on where this is...